sources/packages/api/src/app.institutions.module.ts

```diff
@@ -31,7 +34,10 @@ import { LoggerModule } from "./logger/logger.module";
     InstitutionService,
     BCeIDService,
     UserService,
+    InstitutionLocationService,
     InstitutionControllerService,
+    InstitutionLocationsControllerService,
```

Comment:
This controller service is not used by any of the members of app.institutions.module. Why is it imported here?